### ISSUE TITLE
Test Lexicon based approach (part 2)

### ISSUE THREAD

**Issue by MahdiHajiabadi675** [MahdiHajiabadi675/tourism_dataset#3]
label	Tweets
0	We don't want India to be second Italy .. or going to third stage. Pls follow the instructions given by central gov… https://t.co/T4Yl4B4NG6 
0	@ANI #TabligiJamat programme was from 13 to 15 th March.On 16th March #Delhi govt banned gathering of 50 or more pe… https://t.co/fC6MLNBZLN 
1	@thejmblya I see y’all haven’t tweeted in a while but imma need that $130 I spent on that ticket back cause corona… https://t.co/lpURl8I111 
0	Thank you God! 🙏 https://t.co/sxymuLrAm0 
0	Dear @narendramodi please impose shut down of @JSPLCorporate located in #ballari district. Lakhs of people work in… https://t.co/2CInMLHdCW 
0	@m_preethish @HathwayBrdband @m_preethish file a case against @HathwayBrdband in consumer court and sue the idiots.… https://t.co/LTmZdEUHmj 
1	I’m trying travel but corona itw 🤦🏾‍♂️ 
0	@CNNnews18 @bhupendrachaube Alert.... violence of corona virus lock down should not be allowed to save humanity. 
1	@rohan_166 @Chellaney There are reports where people who were part of the conference hid their travel history and b… https://t.co/TcCcPYMYTW 
0	@RussWildeJr @RichLightShed @byIanThomas at least in respect of the patterns there wasn’t much “impact” - https://t.co/5g2uIwnsnN 
0	@sagarikaghose Let it be. We will deal with it. First of all let us come out of this corona virus. 
0	@stoneybrew It will end in corona 😌😌 
0	"Human dignity, that first instalment, paid up front, by the most vile or able, in the face of #disaster #epidemic… https://t.co/FBmKuPyTKD "
0	These simulations show how to flatten the coronavirus growth curve https://t.co/Ms3YyBoHNU 
0	@teresedau1 @macsween_prue @zalisteggall Imagine where we will be if we don’t turn the climate boat around though-… https://t.co/0Qb9SK9xJ7 
0	Guys add this number on your WhatsApp. It's an official number of government for this Corona updates. #StayHome… https://t.co/77mEXinB7p 
0	So even after a lockdown declared by the govt of india what were those scumbugs doing !!!  And they are now spreadi… https://t.co/ui2mbeE3Zs 
0	@Nigel_Farage If it weren't for the idiots who won't listen it wouldn't have come to this. Only themselves  to blam… https://t.co/XmaoBNznHu 
1	@sunniesmatcha let’s travel somewhere after corona calms down 👉🏼👈🏼 
0	Covid-19: WHO stands by recommendation to not wear masks if you are not sick or not caring for Corona virus patient… https://t.co/Plvasfd7rQ 
1	Empty airport means Corona free. Ready for the sky let’s fly  ✈️ @… https://t.co/uJW8dG19wJ 
0	May God protect us in this Corona pandemic. Plus we have to take precautions also 
0	"my dads got corona symptoms &amp; so him, my step mum, my sister &amp; brother are all self isolating. i’ve not seen them i… https://t.co/XINeGq2ISu "
0	Fatwa to close all the mosques 🤔🤦🙏 https://t.co/iD5YL1z2qu 
1	@narendramodi I request yu to double screen all international traveller from nw on. They should carry health certif… https://t.co/v3iTrlvamr 
1	"The Day After Corona, how many of us will: Go for a Holiday, Movie, travel by Public transport, visit the Mall, go… https://t.co/lUW5J9fWqV "
0	Folks please visit youtube and type in THESE COLORS DONT RUN you will be able to see a complete presentation on a p… https://t.co/kcv5wdPOmw 
0	@HMOIndia please investigate this case and put culprits behind bars as they have put thousands of lives in danger.… https://t.co/RPJ9U6JfP7 
0	"Post-corona world order: What was it not, what will it be https://t.co/sUSuoBRiD0 https://t.co/5SmhS4Fgul "
0	Important: A letter to our customers about the #Coronavirus pandemic and about our temporary closure on April 1st..… https://t.co/iFy0ZmdFlh 
0	WHY? This question needs to be answered. https://t.co/hPzV9xcY4s 
0	"@elonmusk @teslaownersSV @elonmusk @Grimezsz @Tesla Elon, you donated $1mil to plant trees, ¥5mil for Corona virus,… https://t.co/AlkccvvXuI "
1	@1994habit jana: the european tour 2020 postponed due to corona 
0	In the news: #Syria announces first #coronavirus death: https://t.co/xLb59jsQ52 (@newhumanitarian) #COVID19 #Corona 
0	This is a pressure cooker when this cooker blast and spreeding Corona virus in public areas concipresye through the… https://t.co/QpQWbZ6kwm 
1	Over 220 foreigners at Nizamuddin Covid-19 hotspot flouted visa rules - india news - Hindustan Times… https://t.co/WcqP2XSAhA 
0	"Sorry for inconvenience, in earlier version some one pointed an issue, we have edited and sharing again with you.… https://t.co/6Z8FZmPSWM "
1	These are tourist destinations for the wealthy at the time of the outbreak of the new Corona virus… https://t.co/R4wCmcYn8y 
0	Bros how many corona patients have you met? Only you??? https://t.co/E2sTWfyprp 
0	@ANoble63393050 @butty_a @Jay_Beecher Lol to you too. https://t.co/JC9Nu9b5Yy https://t.co/5jlgIbQNIO 
1	"@airvistara i have booked a round trip from pune to delhi during 3th to20th accordingly, due to corona pandemic am… https://t.co/G5vCnXsrjH "
0	@NumanAcar_ @SHO_Homeland @LeslilinkaG @HowardMGordon @PatinkinMandy @CostaRonin @NimratOfficial @MaurySterling… https://t.co/lsQVEO0smh 
0	"Again gone into tangent of VIO weddings. When u spl on one subject, spl on that only, https://t.co/37SZEOBKSz "
0	Is  businessman Kudakwashe Tagwirei and his Sakunda Holdings the answer to the corona virus in 🇿🇼? @daddyhope… https://t.co/K23um6Es6O 
0	I used to be your fun @CisNyakundi  but instead of focusing on important national matters like   @DavidNdii… https://t.co/jOgyUBJHgq 
0	can this corona thing end faster i want to change my bands lah sia 
0	there was a man who was shown via video chat on CNN that said he hasn’t left his house during the outbreak yet he c… https://t.co/ic7Ch6d1xS 
1	"@AliRazaTweets Tableeghi jamaat is not responsible, it's govt. Why people from foreign countries were not checked a… https://t.co/30QQhXAUpL "
0	@AderonkeMedia 😄😄  But how do we sustain the farabaleism state of mind after corona? 
1	"with corona, our tourism economy has stopped. many investors prefer layoffs &amp; aren't ready to give part of their sa… https://t.co/878gZnAcqe "
0	Freedom of speech is what have leant here https://t.co/IBXh4OWFO7 
1	"@sakshijoshii Madam, 1000 of flight were not stopped even after corona virus entered in India  Our health ministry said no worries "
1	*What is Corona virus*  Corona virus is a virus that cannot allow the richest people to travel to another country.… https://t.co/HrO0hY2npp 
1	@sardesairajdeep @drharshvardhan @RahulGandhi Yes we should have banned foreign travel before Corona patient was found in china 
0	TB has highly effective vaccine and medicines available. So that makes it a whole lot different. Keep these analogi… https://t.co/VOeuzQhwfm 
0	"---&gt;  Los Angeles County Officials Plead With Elderly, Those With Health Officials To Stay Home As Corona-virus Cas… https://t.co/rdhjVAUwyJ "
0	@Ang_1k corona made us all act up 😭 
0	Morning @janegarvey1 please help us spread the word about our crowdfunder https://t.co/WALLz6W3a8 We are trying to… https://t.co/QTgIioGbej 
0	@vvoberoi @Swamy39 @Ramy04134006 #howtosaveoureconomy  Nation building is not the work of the government but the du… https://t.co/T10O4di4zl 
0	If this picture is fake .the posterity who remember Corona will also remember Sirdesai for negative role he played… https://t.co/AgjjQKYwUS 
0	If John Smith went round shaking hands in with possible Corona carriers but washed his hands &amp; spoke  of herd immun… https://t.co/M7v931ZJ0M 
0	1520: Smallpox 1620: Bubonic Plague (New England) 1720: Bubonic Plague (France) 1820: Yellow Fever and Cholera  192… https://t.co/VwLVTqD8ZB 
0	"@yassd32k1 @fascinatorfun @2ears2wheels a five year study? isnt corona virus supposed to be new, like from december 2019? "
0	The UAE Government has been proactive in providing the right guideline for Business Continuity in the region.… https://t.co/1TdCVGosGx 
0	"Covid-19: Birth of Corona, a Great Indian Bustard chick in Jaisalmer enthuses bird lovers https://t.co/mq2iW1hUHO "
0	A new zone. I still think we move backwards if death comes calling in our family and friends. https://t.co/tYRM41hQhm 
0	"In today’s show, my guests will discuss how to develop a strong microbiome - to strengthen our immune system and de… https://t.co/KgWtYKlCfv "
0	Lol @SAN_DEEN giving me positive comment. Corona enter your brain? 🤣 
0	"@KTRTRS @TelanganaDGP @GHMCOnline Gud aftrn sir\mam people who r roaming without any reason,dont beat them jst atta… https://t.co/scLGW4PUJ2 "
0	"Together, under our #Tiranga, we stand united, and we will ensure that India emerges stronger, safer, and more pros… https://t.co/I9ljwn0sdP "
0	Please assist me in reporting this ASSHOLE for racism. Fucking hell this is unnecessary. https://t.co/nQhGoW7dXK 
0	He’s IIT/IIMA. it seems they are entitled to be stupid it appears. https://t.co/Ji2WvCO4cJ 
0	social distancing woi https://t.co/Pccv9KBWDn 
0	"I could hear ""ଦେଖିବ ଆସ"" in this dangerously stupid video. Do you also hear it? https://t.co/5mNN7Z1U1B "
0	another day of corona 
0	@TOIIndiaNews #Islamophobicindianmedia only targeting #indianmuslims on #jamat issue but silent over #hindus who ar… https://t.co/64t3PupZC7 
0	"@ANI What if lives are lost due to a bad economy?  Currently, Corona a virus is on a killing spree, in a bad econom… https://t.co/1UNdQn6NNH "
0	"@radioalgerie_ar I have no money i have no food i have no work for issue of corona-virus, beloved brother and sister please help me "
0	"Even Foxnews reporter is yelled at for asking Trump about the lack of Corona Test Kits. Next, instead of talking ab… https://t.co/zgFEoT0TSB "
0	Let's fight together to defeat the Covid-19 by staying at home.  #corona #coronavirus #virus #china #beer… https://t.co/y7VXZ9R9qD 
0	Detailed Listing Of Fund By #Thala Ajith for #Corona Relief   CM Relief Fund : Rs.50L  PM Relief Fund : Rs.50L  FEF… https://t.co/WVcGr7sbvL 
0	"@realDonaldTrump I love you ❤️ so much president Donald J Trump, you are a truly fearless in this time of epidemic… https://t.co/IrJ2gv3Yjk "
0	‘Wrong corona virus’ #COVID2019 #CoronaUpdatesInIndia #lockdown #coronavirus https://t.co/xZ2jYm6vfI 
0	The main reason of these increasing number of Corona Virus cases in past few days is Tablighi Jamat. 
0	@brianstelter @Chloe_Melas WHO did not declare corona virus was a global pandemic until March 11th. WHO said the vi… https://t.co/YwXbXAftDu 
0	@Hbomberguy Any chance or a Corona conspiracy theories measured response? Probably a bit short notice for the amount of research required. 
0	Show off your corona body. 
0	ok corona I’ve had enough https://t.co/kNbUcn6VcM 
0	@LiveasRx @leahmcelrath @dalmatianjaws ‼️Let’s Get Corona Done ‼️ 
0	@OleItumbi While corona vaccine was away..... 
0	Damn you #Corona 
0	Thala Ajith Donated 1.25 Cr For Corona Relief Fund 😍😘  #PerfectCitizenThalaAJITH 😎 https://t.co/gb03vLLTOD 
0	@MohilEr @Gurmeetramrahim @derasachasauda #WorldHealthDay #Covid19 Yes this is right way to fight against Corona 
0	"While the world is grappling with the COVID-19 virus, India’s #BJP government is peddling in hate and weaponizing t… https://t.co/hj0qOkAzdw "
0	@YahooEnt She was having a conversation with her comic friend. It was fun &amp; funny. She’s a comedian &amp; was telling j… https://t.co/6cpoGo1xlt 
0	Is it a creation of Modi government. Why are you exposing past of Congress goverments most recent of one decade long https://t.co/VqJmLytXdj 
0	"Forgot to say that I and @DalalFaisal519 always make plans for breakfast, but we cancel them. Before corona virus w… https://t.co/gvOg7DC390 "
0	Corona makes everything get hard 
0	"Please watch ""Doctor at Queens Hospital in the UK advises on potential lifesaving corona virus breathing technique""… https://t.co/fKjoNXLOhK "
0	"@rajeshtope11 definitely sir, your efforts will make Maharashtra corona free "
0	"His Students Rocking in ""DRONE"" for the Medicinal Purposed for Corona  DONATED Relief Fund in each &amp; Every Sectors… https://t.co/9GoZS4GQOQ "
0	Transatlantik #businessbarometer in times of Corona with The American Chamber of Commerce in Germany e.V. (AmCham G… https://t.co/FWSM23nsmf 
0	"@TheCollectiveQ We patriots in the ""51st US State Germany"" wish the boys of the 'US Army' all the luck in the world… https://t.co/6AcD5rKys3 "
0	This Tweet from @lunadamiano has been withheld in: Germany. 
0	Ready for a shoot :) How's your day going?  #model #modelling #shoot #camera #essex #clacton #clactononsea #seaside… https://t.co/nD8iLwE9An 
0	@RayOrion_ Duh that’s why I ordered a knife that shit laced wit corona no one gon want smoke😂😂💀 
1	@muskateer5 Corona has SCREWED my world tour plan sir:) 
0	Respected  Small information please conduct one day program for rapid kit test on corona virus  for all peoples liv… https://t.co/1oHRCnYUQZ 
0	"Corona, 10 times more dangerous than swine flu, can only be completely prevented with vaccine: WHO… https://t.co/NuceCnDX06 "
0	"@AnilKapoor @rajnathsingh  Dear sir, Please have a look on this, I have tried to write a small poem against corona… https://t.co/n6G3vnQDGR "
1	@ShefVaidya Two actions Not Banning of InteRnationals flight as early as March 1st Allowing Tablighi Jamaat meeting… https://t.co/xnhTtDp1Mz 
0	#corona #covid19  COVID-19 Reports as 14/04/2020 06:17 (UTC)  Confirmed Cases: 1882215  Recovered: 451914  Deaths: 119686 
1	"@airindiain Dear Air India,                                               I have booked a ticket VARANASI to Mumbai… https://t.co/SkWVIGrGUc "
1	"@_sabanaqvi Use some common sense, if you have... Initially incoming passengers from a few #Corona ridden nations w… https://t.co/HfULAEFZMu "
1	@PLL_RBLX   We have our flight on may 3rd from Delhi to budhapest via warsaw  No appointment for visa plus fear of… https://t.co/WNIBAZClUZ 
0	@buthscupoftea Probably a corona in the form of herpes 
0	These people are fighting the real corona ! But still they take risk just to save our lives.  A salute to these rea… https://t.co/KsWiHKSfrb 
1	@palkisu Hoe the fuck did corona get on board American aircraft carriers if it’s made in China? And bitch get yours… https://t.co/mhpktMNa07 
0	"@chrislhayes Why not, Chris? What if China doesn't count people who died from other causes such as stroke, heart di… https://t.co/TYs9or1Kna "
0	@KremlinRussia @RahulGandhi President of Russia I don't understand language but I know humanity have same language… https://t.co/9VhN1HGQq4 
0	Video of collapsed male at WTC result of fatigue; Not related to corona virus - NewsHub #COVID2019… https://t.co/NezWPRiZNY 
0	Honorable PM Modiji ! Doing wondetful job in controlling the corona virus in the country. But for your timely actio… https://t.co/ffZ9iyPmLs 
0	IF you WANT Progressivism to STILL have a REAL VOICE help me get a twitter army to impress Dem Party(NO GP in WY!)… https://t.co/HNlid4oxMe 
0	"@DC_Gurugram @gurgaonpolice @sonalgoelias @cmohry @narendramodi  Respected All, Trust me , Gurugram will Corona /Re… https://t.co/Ly3ePS1zaJ "
0	World is in trouble and facing Corona this man is trying to fire several  cruise missiles in his country at eastern… https://t.co/kX0tzBX5L9 
1	"First corona case was found on 30 Jan 2020,and PM is saying we had been started scanning of corona at airport befor… https://t.co/JYaff9rY2a "
1	I've never flown first class. Is this a real thing or a joke? https://t.co/hxlVL89BPP 
0	"@thismorning @_alicebeer So, so disappointing to see hosts of @ThisMorning failing to categorically debunk, on-air,… https://t.co/oKmJYKaVtS "
0	@motaraaaa Corona peper must have affective here 
0	"@Uber_Support My name is Simon am already a partner with uber in Kochi, this is kindly inform you that this corona… https://t.co/xg9Jp4kyhl "
0	"@HHShkMohd Assalamualikum, Wr are distributing foods in to poor people in India due to corona virus so pls contribu… https://t.co/UAOtLHrwwv "
0	@angus_roxburgh @Peston Corona virus is never going away so it's tiring to keep hearing about which country beat wh… https://t.co/YAqS2mWeEl 
0	Someone really needs to explain to me the correlation between the Corona virus and 5G network 
0	"Live Data Statisitics for COVID-19  India 🇮🇳 Cases: 10541 Deaths: 358 Cured: 1205  Global 🌍 Cases: 1,930,272  Death… https://t.co/95rgyZ1ioc "
0	Get up-to-date information on #COVID19 using the South African Corona WhatsApp line – 0600 123 456      @HealthZA https://t.co/92XKX71w11 
0	May Allah bless her good health and recovery soon AMEEN https://t.co/1T5w2BUQol 
0	"The official name of any political engagement going forward, shall be called post Corona and not post handshake. "
1	"@nileshlog Hi Nilesh, we understand your apprehension to travel due to the outbreak of Corona-virus. However, the c… https://t.co/zMsMoL7xhQ "
0	@Nathani21091749 Lols 😂 corona or not 
0	"@Shobayok1 Babami, good morning. Account is back o. The thing don well from corona 😂😂😂😂 "
0	@EDeCaal @bennyjohnson @BonginoReport He’s a sad little insecure man. Focus on the enemy. The corona virus should be his only concern. 
0	"What the fnk  @UmemeLtd, doing, are they also trying to confuse the corona virus, how are we going to see the presi… https://t.co/q9IpRqS70d "
0	@luxfnbr Nah im too sick wid corona 
1	@MicheleONeilAU @sallymcmanus We have many airlines ( not 2 ) and many brands including REX tiger virgin jetstar QA… https://t.co/7fAxXqyUTr 
0	I GUESS IT’S MY FAULT THEN WE STILL HAVE NO VACCINE AGAINST CORONA HUH 
0	Good speech https://t.co/R6lbBJh0Ic 
0	@I_Am_SyrUp Because of possible Corona bro 
0	That prophet wants his coins y’all 😂 corona or no corona people must give their offering and tithes 
0	Fuck corona I want my life back. I actually miss my job too 
0	671 dead in NY just last night. Smithfield foods in S. Dakota closed it's food facilities after 300 test positive f… https://t.co/zVdQX63V19 
0	Coronavirus Authoritarianism Is Getting Out of Hand  https://t.co/PSK8bOr2kU 
1	our hotel booking for nrs london got cancelled bc of corona https://t.co/Xkqdq4Iw7s 
0	@DrRPNishank @KirenRijiju @cbseindia29 @FitIndiaOff @narendramodi @AmitShah @PMOIndia @HMOIndia @mygovindia… https://t.co/9H9OROvVp3 
0	@xuxisglow @Whitefox_Spry They’re probably getting one soon. But it was canceled due to corona virus 
1	As soon as this corona virus is over who wants to road trip Europe w/ me??? 
0	@DStvNg My DSTV sub will expire tomorrow and I don't have money to sub I need 1 month free Corona package   7040092… https://t.co/vb91NJLsoY 
0	Bees don't catch corona-virus STAY THE FUCK INSIDE 
0	"3 new corona positive reported from Suryapet (Suryapet town - 2, Thirumalagiri - 1).  Suryapet district Total - 23 "
0	"Germany: Police attacked with stones, iron bars while enforcing coronavirus social distance measures… https://t.co/eqpQ7B5AqF "
0	@ANI any measure shifting of corona -ve from  dharavi to other area on temporary basis 
0	@hwngpuppy there's literally idea that we should reopen school's when there's over 100k cases of corona here in ger… https://t.co/hV7X5mk2rd 
1	If  workers are to send home town... CORONA SPL train (The train coaches which are prepared as isolation compartmen… https://t.co/agHzvSoguK 
0	@99freemind @oohisis Or the corona virus symbol. https://t.co/fqlDZXTiFe 
0	@RationalPanic @NigelAWatson @JamieBrysonCPNI Why the hell would we listen to that eejit about corona virus. He has… https://t.co/ryT2aQpahw 
1	"@AUThackeray Dear mr Aditya, if train started for 24 hrs, there will be huge crowd at railway station, so how we ca… https://t.co/tP1nEFRNDF "
1	@TV9Marathi @AUThackeray By this people can get infections of corona. By running train for 24 hours 
0	@MigrationWatch @patel4witham A stupid poll requires a stupid answer so I voted no. Let's flood the country with ev… https://t.co/hIFQBFnAzo 
0	Fight the Corona fear Give the doctors medical gear. #PMDoesNotCare 
1	@Tini_zwitschert I would’ve gone actually. I had found dirt cheap plane tickets from my town &amp;accommodation was alr… https://t.co/4hdrFJppKV 
0	@TheedeeDaFoodie Corona season.. We are stuck at home.. Pls ship food😩 
0	34 people exposed to Corona positive kept in Baldevgarh https://t.co/Ga7mIJlUaL 
1	@goairlinesindia Thanks a lot for help. First airline from whom got proper customer support during this corona time… https://t.co/vLnGjtKX9G 
1	"Between Corona, Graham's travel dates to Paraguay, and mine to France we aren't going to see each other for like 7/12 months 😭 "
0	#Modi ji jara hame utne too doo sir.... Woke up at 1Pm and our #PMModi alredy come up with #Lockdown2.o 🙄… https://t.co/SHwiLvvHZu 
1	"Coronvirus: In relief for stranded Indians, US allows H-1B visa extension https://t.co/ehFakTswYs via @IndianExpress "
1	@Khasaia_ Whole holiday was ruined and I rebooked it for this year because it was such a beautiful resort but corona ruined it 😂 
1	The US government has decided to allow applications by H-1B visa holders for an extended stay in the country  https://t.co/Q4bBhuLxW1 
1	I was planning to travel from Barcelona to Katowice with flight code NDGCQJ on April 17. It was canceled due to Cor… https://t.co/vOVnsh7gq9 
1	@bigdicklis @eunbeaniebaby //sadly you're too far away and there's a Corona stopping me from traveling 
1	Me thinking about the good things that couldve happened if only CORONA didnt exist:  *BrightWin trip to Europe *Son… https://t.co/sgZuiTXyBQ 
0	Marital upside of corona lockdown https://t.co/3S8sGw7Wq6 
1	@IndiGo6E My PNR is BJCIKW . My flight is cancelled due to Covid-19 (which was scheduled on 17th April  2020) So as… https://t.co/coVVGzysLT 
0	@IASassociation @IPS_Association  This pic and the massage of Sardar Patel should be talisman of your fraternity https://t.co/ZHCYAGl8jS 
0	Antibody tests for the coronavirus might not be the 'game-changer'  https://t.co/hgpFmcTcLd 
1	@narendramodi @PMOIndia @HardeepSPuri how can you imagine that 40+ day flight ban will cull Corona. Only vaccine wi… https://t.co/h20LgVmKdM 
1	Aviation Minister announces extension of airport closure as Covid-19 spreads https://t.co/SpjeyZb91r 
0	"Weyyy if only Oil was liquor.....  ""I'd like 100 barrels of Johnnie pls."" ""Yes I'll take it off your hands for $100… https://t.co/q1dAfhhsQB "
1	I’m so proud of my province. Less than 100 COVID deaths. Amazing. And we are the hot spot for travel and tourism. S… https://t.co/HbhYvlVWN7 
0	"Coronavirus: Texas Illness Count Grows To 19,458, 18 More Deaths https://t.co/sDGC6x9Xg1 https://t.co/2HCmkSmZB7 "
0	#COVID19 :| #China is still silent and all the world requests the whole data to be invistigated. I think the owners… https://t.co/kgBnY8QOE1 
0	Your smile is gold *Don't touch it* https://t.co/W6odbhGuzZ  #starlinksatellites  #crudeoil  #420day  #bbrightvc… https://t.co/ThTp2muBGd 
1	Bataan Gov. Albert Garcia last week commissioned idle hotels in the province to make life more comfortable for doct… https://t.co/gi4gEFIft9 
0	Cruzeiro delays cast pay and points to coronavirus crisis as a factor https://t.co/nb3wYUMgwz 
0	Listen im staying Corona free I aint even kissed no one...lmao 
1	@IndiGo6E  My PNR is DKIC6D My flight is cancelled due to COVID-19 (which was scheduled on 12 April  2020) So as pe… https://t.co/XmCbF1Ftte 
0	"Coronavirus:  Kano records 23 new cases, total now 59 https://t.co/G7WsVyZrWD "
0	What behaviour is this Bangladesh? https://t.co/b6svhX8L2c 
1	"Farhat Jamal (@hotelier_19), Chairman of Travel Tourism and Hospitality Committee of the IMC Chamber of Commerce an… https://t.co/nMAS1EQBqQ "
0	Scientists Are Tired of Explaining Why The COVID-19 Virus Was Not Made in a Lab https://t.co/6YbQFUpKfw 
0	@filtremin Coronaviruses are a large family of viruses that are known to cause illness ranging from the common cold… https://t.co/M6HkY30deS 
0	"@narendramodi Modi Sir,  I have a suggestion people who are stuck many places in the india due to lock down, please… https://t.co/2iFll1adSE "
0	"@pushpendraRamji Not until covid19 pandemic is around, we are not #TabligiJamaatVirus . "
0	⚠️🦠 Virus forecast model from US defense team draws attention - https://t.co/f3f90Cb9G4 CLICK to read 👈 Virus BREAK… https://t.co/U3JCbcNYx3 
1	I hope that with this bailout all airlines stop charging for luggage at all!!! https://t.co/Tf54035wfF 
1	"After more than five weeks of the cruise industry announcing plans to voluntarily suspend global operations, three… https://t.co/pOX9gBJxUl "
1	@magazine_42 interviewer Jonas left the #USA because of the #COVID19Crisis. He describes what it's like to travel i… https://t.co/ztW8NmjKoG 
1	“Travelers arriving from overseas don’t have the right to use the airport MRT. Taxis with drivers trained to protec… https://t.co/6VoL7NjI6C 
0	COVID-19: EU launches data sharing platform for researchers https://t.co/BXMf9Dyedb #audio #business 
0	"12-Year-Old Walks 3 Days Amid Lockdown, Dies Just An Hour From Home - NDTV https://t.co/BH4UivZLmw "
0	Trump says WHO didn't share early information about Covid-19. A new report shows that's not the case. || By: Riley… https://t.co/4DJrYB8c5G 
1	"But how do you travel from a Corona hit hotspot and come and demand preferential treatment, you hear us complaining… https://t.co/PiiEXMK25A "
0	petition for a commemorative day slash program slash funeral slash party slash protest in relation to covid19 once all this is over 
0	"That's what happens when country sends essential business out of country,  have to backtrack, how people think year… https://t.co/lzSp8LCET1 "
0	@agencia_andina Your Tweet was quoted in an article by @Agencia_Andina  https://t.co/MvGrlVBKsa 
0	'Remembrance is constantly evolving': This year's Yom Hashoah Holocaust memorials are forced online by the coronavi… https://t.co/D3XOKDNC0i 
0	"@LionelMedia @glennbeck If true, kind of a odd thing to say....even joking. https://t.co/x9GAnE9mtx "
0	Bunmi........come and see your fitness inspirational mumu Governor @bunmola2010 https://t.co/dnzy859cOG 
1	Why is there no Covid-19 screening for passengers arriving at UK airports? https://t.co/fzIqFUtUrb 
1	"@washingtonpost @Jillsey11 People are suing cruise ships for Covid deaths, who will they sue? "
0	What if marijuana ray tambal ani? https://t.co/3SlsYR9EqU 
0	Join us on the 30th April for our 1st webinar discussing The importance of remote operations and digitising operati… https://t.co/NVsqfbn6Wc 
0	@AssholeFrmNY You might have covid-19.... 
1	"Health, and health checks, will dominate all facets of the future international travel experience. https://t.co/6QVJOuYivu "
0	"9 days left until we get to the 100,000 daily Covid promised tests figure in the UK. Latest figures show just betwe… https://t.co/qgubcycXos "
1	@HardeepSPuri @DGCAIndia @airindiain I had flight booking PNR YWDRE. Flights got cancelled due to lock down for Cov… https://t.co/D8Q6WUXA6m 
1	@IndiGo6E  My PNR is  CJJSXW &amp; GE9QFI ...My flight is cancelled due to Covid-19 (which was scheduled on 30April 202… https://t.co/hEWjE70EOk 
0	"Let's do a virtual world tour with New Zealand and Australia, as you head into your evening 🇳🇿🇦🇺  Please help by bu… https://t.co/AoYBrNr006 "
0	COVID-19 Slams Comic Book Industry https://t.co/fxUVZLRgv0 
1	@Andrew_Adonis The airline industry is worth $900 billion to economies ! Millions of jobs and livelihoods are depen… https://t.co/ehUZiEqtRQ 
0	Accused and prosecution want a jury but refused under new COVID legislation in the ACT: Judge alone trials in the t… https://t.co/9uSyXoFlqj 
1	@IndiGo6E My PNR is DGZHKS My flight is cancelled due to Covid-19 (which was scheduled on 19th April  2020) As per… https://t.co/ne6Xm5V1qQ 
0	@BravinYuri Govt is using this Corona virus as an opportunity to  milk Kenyans dry 
1	"Around 50 members of Tablighi Jamaat in Sultanpuri camp, and 20 from Narela, recovered from Covid 19, filled up for… https://t.co/rBUC8C4HMG "
0	During lockdown without humans ❤️ @one_by_two @shufta20 @Iamkavitak @sushant_says @susantananda3 @HiHyderabad… https://t.co/bX9N64bxXi 
0	"Grasping at fake straws. Enough abuse of the people in America, ENOUGH. ONLY RACISTS FOLLOW TRUMP &amp; ITS A RECORDED… https://t.co/Y62AkiatZz "
0	@DAILYNews #DonaldTrump resumes attacks on China over #CoronaVirus 👇👇 https://t.co/lhf8wLjEfX 
0	the cronies are fed https://t.co/cq9abvulx3 
0	nightmare literally a nightmare https://t.co/EkqHXzGhoL 
0	ALL YOU NEED TO  CRUSH COVID-19  Act confidently against COVID-19 with the help of our guidelines and research esta… https://t.co/kDzio67LmD 
0	"@LuisPenny2 @GovMLG @NMDOH @SenateGOP Millions, while the @SenateGOP soaks the taxpayer for 100's of billions?  pag… https://t.co/mGXyl8eGxp "
0	Beijing pressured EU to cancel coronavirus report #SmartNews  https://t.co/IkBsisOzYY 
1	Cathay Pacific looks to increase passenger flights in late June if coronavirus travel restrictions are eased… https://t.co/uC1Jn0OsM4 
0	Don't hold your breath https://t.co/Gw0dpz6RKD 
0	@GOP Americans overwhelmingly believe your leader is an untrustworthy liar. Is he really the best you have? https://t.co/hcyfhpmcUN 
1	"Do you have dementia, Hipocit Nancy? I remember your idiotic comment of racism when Trump shutdown air travel on 1/… https://t.co/5oRQRvIr3K "
0	COVID-19 really distracting us from the fact that the government confirmed aliens exist and they probably know we exist too. 
1	"Day39 HDFC smartbuy has no answers about the refund , their travel partner Cleartrip and HDFC smartbuy joined hands… https://t.co/aWdsXX3Fe5 "
1	"Louisville has canceled their trip to Japan this summer, and Lindsey Duvall is transferring: https://t.co/7oToUBdtqV "
1	"COVID-19: Requesting a voucher vs. refund for flights, hotels and cruises  https://t.co/IlwhISO2jB "
0	"For those of us who participate in #Ramadan, it may feel strange not to go to the mosque or visit family.  But with… https://t.co/NJfP2EZ47B "
1	No one should be selling travel items. We are going nowhere!  we gonna be staying in a house... in the house bored… https://t.co/peH9LajYE3 
0	"I realized as soon as I uttered this to @RealPatterson50 that this was the move, so I'll share it with my timeline.… https://t.co/pi7a7GdNHF "
0	pardon ???????? https://t.co/aRhNIsQFhc 
0	They want to keep you ignorant.  Important to watch. Make your own mind up. https://t.co/EHGiLrfspX 
0	"Kerala Update: With no #COVID19 positive case in Thiruvananthapuram currently, the city has been removed from the list of hotspots. "
0	Boom goes the dynamite. https://t.co/reIWTMiEGd 
1	"@airindiain Hi,I booked my flight (PNR:HEXJK) on 21.03.2020 from MEL to DEL for 30.03.2020. Unfortunately, flight g… https://t.co/OEbeuh3oaD "
0	US oil prices sink as coronavirus fuels storage fears Traders fear WTI crude could again turn negative as pandemic… https://t.co/SQEyfkVNj4 
0	"The health, safety and wellbeing of all who work for and with us is of paramount importance during this pandemic. W… https://t.co/YbeDWRxf6q "
0	If you love The Goonies you NEED to watch this.  The first time I watched The Goonies was on my very first date eve… https://t.co/jrRpBcCVyQ 
0	@c_legoz1204 @slpng_giants @RealCandaceO @DiamondandSilk She's getting paid to:  • claim she won't get a vaccine •… https://t.co/4lJ52kHans 
0	"@ayotv_ @BOpinion1 @bopinion @bunge_tz Once a country lifts lockdown orders, provided there’s still no viable vacci… https://t.co/lhfADrWoso "
0	Coronavirus: Maharashtra govt to send 100 buses to bring back students stuck in Kota   https://t.co/klWsMUEb3z 
0	"Greater thread here. It’s one of the most fair-minded accounts I’ve read of Japan’s neighborhood associations, and… https://t.co/cFk64Dd75t "
0	@shanetold Do you think the 20th anniversary tour will be pushed back again? Just curious due to some things I’ve r… https://t.co/OydGq47g2H 
1	@pratikpattana15 Your flight is impacted due to COVID-19 and we're in a process to create credit shell. You will be… https://t.co/EF4rC8TpjZ 
1	Dear AirAsia related with COVID19 Please help us with the Credit in my BIG ID account to fly again with AirAsia Aft… https://t.co/WJhd4OK68A 
0	A superhero before being one was cool! @alastanford #ILookLikeASurgeon Stand for something or fall for everything! https://t.co/Ekd0hXvV5V 
1	…  program to aid airlines and cargo carriers has been relatively trouble … .         Mnuchin also announced that a… https://t.co/zIaPI06gfl 
0	"Many of us knew this for a while now, why has #Cuomo so many #COVID19 deaths (not really)... #coronavirus https://t.co/SmJ4H19AbL "
0	"Some are stuck somewhere, some wish they were somewhere  Now we are counting days, to go back to our regular ways W… https://t.co/vgqRaVVdsu "
0	they really talkin about the time obama wore his tan suit in the comments lmaooo https://t.co/NhHEXlO7Jt 
0	"It's time to STOP letting fascist governors and ""experts"" dupe us into thinking that #stayingathome is #savingalife… https://t.co/Bsap8gCf6o "
0	@Akonedo COVID19 SHOULD NEVER BEEN SEEN AS COMMON COLD!!!!!!!!!!!!!! 
1	You could come back with SARS-CoV-2 after a trip to Woolies https://t.co/mk0PJWtMBi 
1	"@IndiGo6E My flight got cancelled because of COVID-19.  PNR WER83Y, I got something called ""IndiGo credit shell"" wh… https://t.co/V9aDfn8MWR "
0	"Biden calls for massive COVID-19 stimulus to start making big, structural change https://t.co/WifHALq7YY "
0	"Please for your health join us onlne on Mondays, Tuesdays &amp; Fridays at 11am &amp; 430pm!!!! #FitnessMotivation… https://t.co/nl0OjKYcq5 "
1	"Airbus to resume work at Brookley on Wednesday, but raises international financial concerns | Coronavirus COVID-19 … https://t.co/Vnn8Zb2BRB "
0	Yes agree we look after individuals and outcomes not outputs. This is time and task and goes against good care prac… https://t.co/8gVnUr6jEP 
0	Covid Toes?? Subhanallah only once covid got to the states did it decide to mutate into some non-flu like disease.… https://t.co/N2mNwDkcRs 
1	"JetBlue will require passengers to wear masks, other airlines will provide them for travelers amid coronavirus… https://t.co/nNNCCaVgeB "
1	"#BREAKING: Scandinavian Airlines lays off 5,000 staff members due to the #coronavirus crisis.  Source:… https://t.co/edCgugUk8Q "
0	@MundaArjun Where everyone is fighting against this desease COVID 19 and it is said that salary must be given timel… https://t.co/SxjwkAwolq 
1	#TravelNews #COVID19 Update United Airlines to introduce new disembarking measures to maintain social distancing on… https://t.co/1uSVs5FrQj 
0	"To cover up the truth, why else. So sick of their games https://t.co/BsKjd6vGai "
1	Word is most of the airlines now require mask inflight. So what are you supposed to do when the flight attendant ha… https://t.co/XW9iboSXkb 
0	#SATnews @shrimanji @PradeepKTaneja @PrashantTandy @PDChina @XHNews @SBSNews @SBSHindi @SBSPunjabi @BBCWorld… https://t.co/TS4UbJo5ka 
0	“We refuse to risk even one Juggalo life” is my favorite moment of covid so far 
0	"Coronavirus in Mumbai: After chaos outside liquor shops, Section 144 imposed in the city till May 17… https://t.co/X4mA2KW66Z "
0	"@JamesWe95821875 @CarwashElmwood @JeffreyGuterman @realDonaldTrump Please don't think that Covid 19 is not serious,… https://t.co/JIa1HjneH9 "
1	Covid-19: Qantas Group Extends Flight Cancellations through July.  @Qantas  https://t.co/vw36Mt4t4u https://t.co/KTamaFyXR3 
1	Coronavirus: Delta says airline will push on despite billionaire selling off stake in company https://t.co/OKA6XLg7hd 
1	"@PRIYANK09214217 Sir, your flight is cancelled due to COVID-19 travel restrictions. We have protected your amount i… https://t.co/bpRnb6Ak0P "
0	His fat ass hasn’t given up I see smh https://t.co/MTS5NVSZCl 
0	RDC Gulu District Major Santo Lapolo Okot: Gulu is a tragic area in terms of communication. We so far have one case… https://t.co/TTDA6yfepw 
0	"https://t.co/zZMMROLOv9 ASIAN stocks rose on Tuesday, following a late Wall Street rally as governments eased coron… https://t.co/X2q2FLLUDz "
1	Desert island! Video tour of Koh Phi Phi on lockdown. #thailand #covid19 https://t.co/zxsDEflWNA 
1	"Very worse for me with this Corona,my wife was to travel a day before the Nairobi lockdown from ushagoo, I have bec… https://t.co/K98zKhEbq0 "
